stubs/netaddr/netaddr/ip/iana.pyi

```diff
@@ -0,0 +1,45 @@
+from _typeshed import Incomplete
+from collections.abc import Callable, Mapping
+from typing import TextIO
+from typing_extensions import TypeAlias
+from xml.sax import handler
+from xml.sax.xmlreader import XMLReader
+
+from netaddr.core import Publisher, Subscriber
+from netaddr.ip import IPAddress, IPNetwork, IPRange
+
+_IanaInfoKey: TypeAlias = IPAddress | IPNetwork | IPRange
+
+IANA_INFO: dict[str, dict[_IanaInfoKey, dict[str, str]]]
+
+class SaxRecordParser(handler.ContentHandler):
+    def __init__(self, callback: Callable[[Mapping[str, object] | None], object] | None = ...) -> None: ...
+    def startElement(self, name: str, attrs: Mapping[str, object]) -> None: ...
+    def endElement(self, name: str) -> None: ...
+    def characters(self, content: str) -> None: ...
+
+class XMLRecordParser(Publisher):
+    xmlparser: XMLReader
+    fh: Incomplete
+    def __init__(self, fh: Incomplete, **kwargs: object) -> None: ...
+    def process_record(self, rec: Mapping[str, object]) -> dict[str, str] | None: ...
+    def consume_record(self, rec: object) -> None: ...
+    def parse(self) -> None: ...
+
+class IPv4Parser(XMLRecordParser): ...
+class IPv6Parser(XMLRecordParser): ...
+class IPv6UnicastParser(XMLRecordParser): ...
```

Comment:
Unlike in the superclass, the `process_record()` method in these classes never returns `None`, so we should override it

```suggestion
class IPv4Parser(XMLRecordParser):
    def process_record(self, rec: Mapping[str, object]) -> dict[str, str]: ...

class IPv6Parser(XMLRecordParser):
    def process_record(self, rec: Mapping[str, object]) -> dict[str, str]: ...

class IPv6UnicastParser(XMLRecordParser):
    def process_record(self, rec: Mapping[str, object]) -> dict[str, str]: ...
```